packages/js/number/src/index.ts

```diff
@@ -65,6 +80,7 @@ export function formatValue( numberConfig, type, value ) {
 		case 'number':
 			return numberFormat( { ...numberConfig, precision: null }, value );
 	}
+	return null;
```

Comment:
Added `return null;` here to match the JSDoc return type. Otherwise, it will return`undefined`.